src/Utility/Instrumentation/Constants.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace OpenAI.Instrumentation;
+
+internal class Constants
+{
+    // follows OpenTelemetry GenAI semantic conventions:
+    // https://github.com/open-telemetry/semantic-conventions/blob/main/docs/gen-ai
+
+    public const string ErrorTypeKey = "error.type";
```

Comment:
Why don't we have otel APIs that operate on UTF8 strings?

Reply:
OTel allows unicode attribute names and uses unicode strings on the wire format.